api/v1/auxilliary.yaml

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
Let's put auxiliary.yaml in /interfaces because that's where the other interface is (and where other interfaces will go).

Reply:
done